dotcom-rendering/src/web/components/TextBlockComponent.tsx

```diff
@@ -35,33 +35,39 @@ const isOpenQuote = (t: string): boolean => {
 	].includes(t);
 };
 
-const isLongEnough = (html: string): boolean => {
-	// Only show a dropcap if the block of text is 200 characters or
-	// longer. But we first need to strip any markup from our html string so
-	// that we accurately measure the length that the reader will see. Eg. remove
-	// link tag html.
+const stripHtmlFromString = (html: string): string => {
 	// https://stackoverflow.com/questions/822452/strip-html-from-text-javascript is
 	// a good discussion on how this can be done. Of the two approaches, regex and
 	// DOM, both have unikely failure scenarios but the impact for failure with DOM
 	// manipulation carries a potential security risk so we're using a regex.
-	return html.replace(/(<([^>]+)>)/gi, '').length >= 199;
+	return html.replace(/(<([^>]+)>)/gi, '');
 };
 
-const decideDropCapLetter = (html: string): string => {
-	const first = html.substr(0, 1);
+const getDropCappedSentence = (html: string) => {
```

Comment:
It felt as though the drop cap and the rest of its sentence is coupled, so tried to keep it as such in 1 method.